tests/testSuspectMasking.py

```diff
@@ -0,0 +1,138 @@
+#
+# LSST Data Management System
+# Copyright 2008, 2009, 2010 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+import unittest
+
+import numpy as np
+
+import lsst.utils.tests
+import lsst.afw.image as afwImage
+import lsst.afw.geom as afwGeom
+import lsst.afw.table as afwTable
+import lsst.ip.isr as ipIsr
+
+class IsrTestCases(lsst.utils.tests.TestCase):
+    def setUp(self):
+        ampInfoShema = afwTable.AmpInfoTable.makeMinimalSchema()
+        ampInfoCat = afwTable.AmpInfoCatalog(ampInfoShema)
+        ampInfo = ampInfoCat.addNew()
+        ampInfo.setRawBBox(afwGeom.Box2I(afwGeom.Point2I(-5, 7), afwGeom.Extent2I(53, 104)))
+        ampInfo.setSuspectLevel(25000)
+        self.ampInfo = ampInfo
+        self.isrTask = ipIsr.IsrTask()
+
+    def tearDown(self):
+        self.ampInfo = None
+        self.isrTask = None
+
+    def testBasicMasking(self):
+        """Test that masking works
+        """
+        maxVal = 32000
+        fracSuspect = 0.3
+        suspectLevel = maxVal*(1 - fracSuspect)
+
+        bbox = self.ampInfo.getRawBBox()
+        self.ampInfo.setSuspectLevel(suspectLevel)
+        maskedImage = makeRampMaskedImage(bbox, 0, maxVal)
+        imArr = maskedImage.getImage().getArray()
+        desSetArr = np.where(imArr >= suspectLevel, 1, 0)
```

Comment:
Why are you doing `foo=where(condition,1,0)` here instead of just `foo=condition`? The latter gives you the boolean array directly.